bftengine/src/bcstatetransfer/BCStateTran.hpp

```diff
@@ -319,6 +319,8 @@ class BCStateTran : public IStateTransfer {
 
  private:
   void loadMetrics();
+  std::chrono::seconds last_dump_time_;
+  std::chrono::seconds dump_interval_in_sec_;
```

Comment:
Add metrics context to the dama members' names.

Reply:
Done